<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mutiRCDialog</class>
 <widget class="QWidget" name="mutiRCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>500</width>
     <height>345</height>
    </rect>
   </property>
   <property name="title">
    <string>遥控器校准</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开关</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>纵向</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>横向</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>航向</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>升降</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>100</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>接收机类型</string>
    </property>
    <widget class="QRadioButton" name="radioButton_PWM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PWM</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_STU">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>STU</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_DSM2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DSM2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_S_BUS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>S-BUS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_X_BUS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X-BUS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rc_receiveset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rc_receiveask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_rccali">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>校准开始</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_switch">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>25</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ele">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ail">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rud">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_col">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
